platform-ui-skin/src/main/webapp/skin/less/platform/portlets/extensions/login.less

```diff
@@ -16,24 +16,55 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 @import "../../../variables.less";
 @import "../../../mixins.less";
-@import "forgot-password.less";
-//========================================
+
+body, html, .VuetifyApp, .v-application, .v-application--wrap, .v-main, .v-main__wrap {
+  height: 100%;
+  max-height: 100%;
+  min-height: 100%;
+}
 body {
 	background-color: @baseBackgroundDefault;
 	font-family: @baseFontFamily;
 	font-size: @baseFontSize;
 	color: @textColor;
 }
+.VuetifyApp .v-application .white-background {
+  background-color: #FFFFFF !important;
+}
+#mainAppArea .v-main__wrap {
+  display: flex;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+.VuetifyApp .v-btn--outlined {
+  border: thin solid currentColor !important;
+}
+.login-button, .register-button {
+  width: 222px !important;
+  max-width: 100% !important;
+}
+.v-input legend {
+  width: 0;
+}
+
+.brandingLogo {
+  bottom: 3%;
+  right: 3%;
+  position: fixed;
+}
+
+/*========= FIXME : To Delete Once MIP Finished. Kept for Backward Compatibility ==========*/
```

Comment:
Should be deleted for merging the PR ?

Reply:
You are right, this CSS Style is useless now that almost all pages have been reworked. Knowing that the original MIP has been splited (oAuth Forms not to be reworked in this MIP anymore), there was a chance that it's still used. I did a check on oAuth forms and it seems not used there.

Thanks for the feedback.